squash/docs/overview/_index.md

```diff
@@ -212,11 +212,19 @@ You should see a screen similar to the one below.
 
 The Next step is to tell the Squash extension the location of the `squashctl` CLI.
 
-Open **Settings** (File > Preferences > Settings) on Visual Studio Code, and then search for Squash. Once you find Squash, change the value of Path to point to the location of the CLI.
+Open **Settings** (Code > Preferences > Settings) on Visual Studio Code, and then search for Squash. Once you find Squash, change the value of Path to point to the location of the CLI.
 
 ![config](../images/vs-code-config-squash.png)
 
 If the target process was compiled from a different sourcepath, you should also [specify the remote path](/configuration/#source-code-mapping).
+In this quick start, if you choose go example2 app, you have to set “Squash: Remote Path” in VSCode User Settings.
+Please set "/home/yuval/go/src/github.com/solo-io/squash/contrib/example/" to “Squash: Remote Path”.
```

Comment:
I'd flip the wording of this sentence around and use markdown code block, i.e.,

Please set `Squash: Remote Path` to `/home/yuval/go/src/github.com/solo-io/squash/contrib/example/`.

Reply:
I understand. Thank for your review!